src/main/java/seedu/progresschecker/logic/commands/SortCommand.java

```diff
@@ -7,6 +7,7 @@
  */
 public class SortCommand extends UndoableCommand {
     public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_ALIAS = "so";
```

Comment:
Might need to reconsider formatting of all alias for consistency? (Not asking for changes in this PR however)